src/util/rebind.js

```diff
@@ -24,4 +24,30 @@
         return target;
     };
 
+    function capitalizeFirstLetter(str) {
+        return str[0].toUpperCase() + str.slice(1);
+    }
+
+    /**
+     * Rebinds all the methods from the source component, adding the given prefix. An
+     * optional exclusions parameter can be used to specify methods which should not
+     * be rebound.
+     */
+    fc.util.rebindAll = function(target, source, prefix, exclusions) {
```

Comment:
Could the exclusions be made an array or var-args? I can see valuable use cases for both.